internal/storage/cache/cache.go

```diff
@@ -0,0 +1,66 @@
+package cache
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"go.flipt.io/flipt/internal/cache"
+	"go.flipt.io/flipt/internal/storage"
+)
+
+var _ storage.Store = &Store{}
+
+type Store struct {
+	storage.Store
+	cacher cache.Cacher
+}
+
+func New(store storage.Store, cacher cache.Cacher) storage.Store {
+	return &Store{store, cacher}
+}
+
+func (s *Store) SetCache(ctx context.Context, key string, value any) error {
```

Comment:
```suggestion
func (s *Store) setCache(ctx context.Context, key string, value any) error {
```

could we make these helper methods unexported?